artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -294,6 +294,14 @@
 
    private String internalNamingPrefix = ActiveMQDefaultConfiguration.getInternalNamingPrefix();
 
+   private boolean criticalAnalyzer = ActiveMQDefaultConfiguration.getCriticalAnalyzer();
+
+   private boolean criticalAnalyzerHalt = ActiveMQDefaultConfiguration.getCriticalAnalyzerHalt();
+
+   private long criticalAnalyzerTimeout = ActiveMQDefaultConfiguration.getCriticalAnalyzerTimeout();
+
+   private long criticalAnalyzerCheckPeriod = 0; // non set
```

Comment:
@michaelandrepearce nope.. because the value will only be determined during runtime (when getCriticalAnalyzerCheckPeriod is called.